hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -807,6 +810,57 @@ public RaftPeerId getRaftPeerId() {
     return this.raftPeerId;
   }
 
+  @VisibleForTesting
+  public RaftPeerId getRaftLeaderId() {
+    return this.getRaftLeader() == null ? null : this.getRaftLeader().getId();
+  }
+
+  @VisibleForTesting
+  public String getRaftLeaderAddress() {
+    if (this.getRaftLeader() == null) {
```

Comment:
Perhaps consider using a local variable to hold the `RaftPeer` with a single call to `getRaftLeader()` at beginning of method.  Then using this var through rest of method instead of repeated calls to `getRaftLeader()`.